openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/AppRepository.java

```diff
@@ -216,6 +216,14 @@ public AppRunRecord getLatestAppRuns(UUID appId) {
     return JsonUtils.readValue(json, AppRunRecord.class);
   }
 
+  public AppRunRecord getLatestAppRunsAfterStartTime(UUID appId, long startTime) {
+    String json = daoCollection.appExtensionTimeSeriesDao().getLatestAppRun(appId, startTime);
+    if (json == null) {
+      throw new UnhandledServerException("No Available Application Run Records.");
```

Comment:
can we throw a better error than unhandled server exception
this give 503 error back essentially causing the UI to throw not a clear back to user.
What we want UI to behave here.

Reply:
also we introduced ERROR_TYPE so that UI can handle those errors better. Lets use that specific error

Reply:
updated @harshach